examples/storage_pool.go

```diff
@@ -10,7 +10,7 @@ func main() {
 
 	var (
 		ClientOV     *ov.OVClient
-		storage_pool = "<storage_pool>"
+		storage_pool = "CPG-SSD"
```

Comment:
Its better to not to hardcode to maintain code standards

Reply:
done